zh_CN/commit_and_prs.md

```diff
@@ -1,22 +1,147 @@
-这将包括代码提交信息和 PRs 的规范：（正在完成..）
+# 代码提交与合并
+
+绝大多数的 UKUI 项目使用 git 进行代码管理，使用 github 进行代码的分发与合作开发。为了便于所有人都可以有序的参与 UKUI 的所有项目，我们希望能够约束 UKUI 项目的代码提交记录、冲突合并和 PRs 创建等各种行为。
```

Comment:
The case of `github` should be consistent. I recommend using `Github`, what's your insights? 😄

Reply:
Yeah, it would be better